.github/workflows/gke.yaml

```diff
@@ -20,19 +20,6 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
```

Comment:
I think we can also remove this line so it is explicit that we are not doing anything from any line of code?

Reply:
We need to check out to get access to master version of the `.github/get-kubeconfig.sh` and helm chart.

Reply:
oh I miss that, you are right.